elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -57,7 +57,7 @@ async def verify_attached_bugs(runtime: Runtime, verify_bug_status: bool, adviso
 
         # bug.is_ocp_bug() filters by product/project, so we don't get flaw bugs or bugs of other products or
         # placeholder
-        non_flaw_bugs = {b for b in bugs if b.is_ocp_bug()}
+        non_flaw_bugs = [b for b in bugs if b.is_ocp_bug()]
```

Comment:
This was making the unit tests flaky. Possible duplicates have been excluded above this step